vuu-ui/packages/vuu-shell/src/theme-provider/ThemeProvider.tsx

```diff
@@ -48,27 +62,30 @@ const createThemedChildren = (
 };
 
 interface ThemeProviderProps {
-  children: ReactElement;
+  children?: ReactElement;
```

Comment:
I think this should be required - it doesn't make sense to have an empty ThemeProvider